component/custom/custom.go

```diff
@@ -0,0 +1,57 @@
+package custom
```

Comment:
Copyright statements are missing. Run "copyright_fix.sh" from workspace root to fix all required files.

Reply:
"./copyright_fix.sh " is the command to use.